### ISSUE TITLE
APSIM root model

### ISSUE THREAD

**Issue by sarchontoulis** [APSIMInitiative/ReferencePanel#82]
As part of a project supported by ISU and AI, we are current working on testing the APSIM NG root module using data from Iowa. we will have our first meeting March 12 to review existing approach, come up a list of modeling concepts for potential implementation/testing and identify additional datasets or scientists. The focus of the meeting will be on the simulation of root mass, root nitrogen, and root to shoot ratio.

**Comment by sarahcleary** [comment]
@uqghamme; @kchenu @ElvisElli 

@APSIMInitiative/reference-panel and @APSIMInitiative/steering-committee - FYI

**Comment by sarahcleary** [comment]
@APSIMInitiative/soil

**Comment by sno036** [comment]
Thanks for the information @sarchontoulis. Will there there be inclusion of modelling location (e.g. depth penetration) and turnover? @hut104 may remember some thinking on the depth question w.r.t. soil constraints in a Filling the Research Gap project several years ago - that may be of assistance in your discussions.

And just one PS - don't forget perennial systems!!

**Comment by sarchontoulis** [comment]
@sno036 yes, we plan to make a start with the root/shoot ratio and root N and then we will move into decomposition and rooting depth, for which we will need SWIM! BTW, how does AgPasture NG handle root/shoot ratio response to N stress? If I remember well, there was some functionality in the classic version. We can learn a lot from perennial systems!
